cli/pcluster/config/validators.py

```diff
@@ -286,7 +292,8 @@ def fsx_storage_capacity_validator(section_key, section_label, pcluster_config):
     fsx_section = pcluster_config.get_section(section_key, section_label)
     storage_capacity = fsx_section.get_param_value("storage_capacity")
     deployment_type = fsx_section.get_param_value("deployment_type")
-
+    storage_type = fsx_section.get_param_value("storage_type")
+    per_unit_storage_throughput = fsx_section.get_param_value("per_unit_storage_throughput")
```

Comment:
E.g. first check can be separate by the other if/else:
```python
    if fsx_section.get_param_value("fsx_fs_id") or fsx_section.get_param_value("fsx_backup_id"):
       # comment
       return errors, warnings
```

Reply:
Done!